sdk/core/Azure.Core/samples/Configuration.md

```diff
@@ -103,6 +103,55 @@ SecretClientOptions options = new SecretClientOptions()
 };
 ```
 
+Another customization that is possible is to easily configure what exceptions or status codes 
+are retried. By default, this is configured in the `ResponseClassifier` type, but it can be 
```

Comment:
```suggestion
If you only need configure which exceptions or status codes are retried, this is also supported.
By default, this is configured in the `ResponseClassifier` type, but it can be 
```